plugin/storage/cassandra/factory.go

```diff
@@ -143,12 +405,21 @@ func (f *Factory) Initialize(metricsFactory metrics.Factory, logger *zap.Logger)
 	}
 	f.primarySession = primarySession
 
+	// After creating a session, execute commands to initialize the setup if not already present
+	if err := f.InitializeDB(primarySession); err != nil {
```

Comment:
what happens if the schema already exists?

Reply:
All the queries use "IF NOT EXIST" clause, which prevents it from altering the already existing schema.